### PR TITLE
Make sure go buttons are disabled until ticks start. fix #81

### LINKED ISSUE #81: All go buttons should have "Disable until ticks start"
We still have lots of models that don't make use of the "Disable until ticks start" checkbox to prevent users from pressing "go" before "setup".

### PR DESCRIPTION
**Pull request by Idloj**

### CODE REVIEW

**Comment by nicolaspayette** [comment]
While I appreciate the effort, it may take a while before this gets merged. I just cannot blindly merge changes to 320 models and will need to carefully check them one by one. There might not be time for that before 5.2.1. Also, as briefly discussed on #81, I would have preferred at least a partially automated solution.

In the future, it would be better to submit much (much!) smaller pull requests. They will stand a better chance of getting merged quickly.

**Comment by Idloj** [comment]
You want me to make 320 different branches and 320 different PRs?

File: Sample Models/Chemistry & Physics/Heat/Unverified/Heat Diffusion.nlogo [hunk 1]
```diff
@@ -213,7 +213,7 @@ alpha
 alpha
 0.0010
 10.0
-10
+0.8418
```
Comment (SethTisue) [inline Sample Models/Chemistry & Physics/Heat/Unverified/Heat Diffusion.nlogo]: what happened here?

**Comment by SethTisue** [comment]
I eyeballed all of the diffs and they LGTM, with one exception, commented above.

**Comment by Idloj** [comment]
Hold on with this, I missed the 3D models

**Comment by SethTisue** [comment]
there are some diffs like `0.005` vs `0.0050` that I don't know how to account for. this may be a previously unknown platform difference in how doubles get printed, or something. (it's also possible we've run into this before and I've just forgotten)

**Comment by nicolaspayette** [comment]
Merged as part of #167.

https://travis-ci.org/NetLogo/models/builds/94507385 shows the risk in bulk manual updates like this. Those 12 models (well, 11: Pac-Man was saved by https://github.com/NetLogo/NetLogo/issues/892) would have ended with non-clickable **go** buttons.

Still, in the end, your work turned out to be very useful, @Idloj. Many thanks!

**Pull request merged**